README.md

```diff
@@ -335,18 +336,18 @@ ShaderModule shader1;
 Result result = device.createShaderModule(&createInfo, allocator, &shader1);
 if (result.result != VK_SUCCESS)

```

Comment:
We should use `vk::Result::eSuccess`, instead of `VK_SUCCESS` here and in the snippets below.